gen-ai/orchestrator-server/src/main/python/tock-llm-indexing-tools/index_documents.py

```diff
@@ -275,19 +315,51 @@ def normalize_opensearch_index_name(namespace: str, bot_id: str, index_session_i
 
     return normalized
 
-if __name__ == '__main__':
-    start_time = datetime.now()
-    formatted_datetime = start_time.strftime('%Y-%m-%d %H:%M:%S')
 
+# Configure logging
+def setup_logging(cli_args):
+    log_format = '%(levelname)s:%(module)s:%(message)s'
+    # Create log directory if it doesn't exist
+    log_dir = Path('logs')
+    log_dir.mkdir(exist_ok=True)
+
+    log_file_name = log_dir / f"index_documents_{datetime.now().strftime('%Y%m%d_%H%M%S')}.log"
```

Comment:
It should be documented in the README.md that we now have logs output in this folder, thanks for adding it 👍️